src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -223,23 +223,24 @@
             </field>
             <field>
                 <id>proxy.forward.sslbump</id>
-                <label>Enable SSL mode</label>
+                <label>Enable SSL inspection</label>
                 <type>checkbox</type>
                 <help><![CDATA[
-                  Enable sslbump mode,
-                  which makes the proxy act as a man in the middle between the internet and your clients.<br/>
+                  Enable SSL inspection mode,
+                  which allows to log HTTPS connections information, such as requested URL and/or
+                  make the proxy act as a man in the middle between the internet and your clients.<br/>
                   Be aware of the security implications before enabling this option.
-                  <br/><br/>
-                  Transparent HTTP proxy needs to be enabled and you need nat rules to reflect your traffic
-                  for this feature to work.<br/>
-                  <a href="/firewall_nat_edit.php?template=transparent_proxy&https=1"> Add a new firewall rule </a>
```

Comment:
I don't think there is a way around adding it back until we have a better solution